src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,84 @@
 package mate.academy
 
+import mate.academy.model.Color
```

Comment:
The `ColorSupplier` class is not imported in this file. Ensure you import it to avoid runtime errors when generating random colors for figures.